spec/models/loader_orchestrate_spec.rb

```diff
@@ -16,22 +16,29 @@ def load_base_policy path
     RootLoader.load 'rspec', policy_path(path)
   end
 
-  def load_policy_update path
-    version = PolicyVersion.new.tap do |version|
-      version.policy = resource_policy
-      version.role = role_user_admin
-      version.policy_text = File.read(policy_path(path))
-      version.perform_automatic_deletion = perform_automatic_deletion
-      version.delete_permitted = delete_permitted
-      version.update_permitted = update_permitted
-      version.validate
-      expect(version.errors.to_a).to eq([])
-      expect(version.valid?).to be_truthy
-      version.save
-    end
-    Loader::Orchestrate.new(version).tap do |loader|
-      loader.load
-    end
+  def policy_version path
```

Comment:
Name suggestion: `save_policy`.

The whole "version" naming was a inherited mistake that we should avoid spreading.